### PR TITLE
Remove $traceurRuntime.toProperty

### PR DESCRIPTION
**Pull request by arv**
Change SymbolTransformer to only transform typeof expressions and
make it on by default.

Also clean up runtime that used to depend on symbol toProperty

### CODE REVIEW

**Comment by arv** [comment]
@johnjbarton PTAL

File: src/runtime/runtime.js [hunk 1]
```diff
@@ -29,10 +29,8 @@
   var $defineProperties = $Object.defineProperties;
   var $defineProperty = $Object.defineProperty;
```
Comment (johnjbarton) [inline src/runtime/runtime.js]: Looks like this one is also obsolete.

**Comment by johnjbarton** [comment]
LGTM

File: src/runtime/runtime.js (hunk 1 above)
Comment (arv) [inline src/runtime/runtime.js]: `$defineProperties` is not used. I assume that you just accidentally commented on the wrong line?

**Pull request closed without merging**